Tests/SentryTests/SentrySDKTests.swift

```diff
@@ -488,6 +488,14 @@ class SentrySDKTests: XCTestCase {
         XCTAssertEqual(SentrySDK.getAppStartMeasurement(), appStartMeasurement)
     }
     
+    func testIsEnabled() {
+        XCTAssertFalse(SentrySDK.isEnabled)
+        SentrySDK.setCurrentHub(fixture.hub)
```

Comment:
`m`: I think we should rather test this with public methods. Something like this should cover multiple scenarios

```
    func testIsEnabled() {
        XCTAssertFalse(SentrySDK.isEnabled)
        
        SentrySDK.capture(message: "message")
        XCTAssertFalse(SentrySDK.isEnabled)
        
        SentrySDK.start { options in
            options.dsn = SentrySDKTests.dsnAsString
        }
        XCTAssertTrue(SentrySDK.isEnabled)
        
        SentrySDK.close()
        XCTAssertFalse(SentrySDK.isEnabled)
        
        SentrySDK.capture(message: "message")
        XCTAssertFalse(SentrySDK.isEnabled)
        
        SentrySDK.start { options in
            options.dsn = SentrySDKTests.dsnAsString
        }
        XCTAssertTrue(SentrySDK.isEnabled)
    }
```